nm/src/nm.c

```diff
@@ -418,8 +418,10 @@ _j4status_nm_device_update(J4statusPluginContext *context, J4statusNmSection *se
 
             value = j4status_format_string_replace(context->formats.up_wifi, _j4status_nm_format_up_wifi_callback, &data);
 
-            if ( data.ssid != NULL )
+            if ( data.ssid != NULL ) {
                 g_variant_unref(data.ssid);
+                data.ssid = NULL;
+            }
```

Comment:
`data` is a local variable and will not survive the `case` block so there is no need to set it to `NULL`.
Did you hit any issue here? If so, it’s a good thing to say it in the commit message and explain why.
It would definitely need its own commit if you really need that.